### DISCUSSION TITLE
Is there any way to remove this?

### DISCUSSION THREAD

**Discussion by letkan** [Motion-Project/motionplus#163] [General]
![1720459537](https://github.com/Motion-Project/motionplus/assets/9017461/68357874-49fd-4746-a916-c029eabd5f3e)

**Comment by Mr-Dave** [comment]
Use [stream_preview_ptz](https://motion-project.github.io/motionplus_config.html#stream_preview_ptz) or create your own page / edit the default and use [webcontrol_interface](https://motion-project.github.io/motionplus_config.html#webcontrol_interface) with the option of `user` and specify your page in [webcontrol_html](https://motion-project.github.io/motionplus_config.html#webcontrol_html)

**Comment by letkan** [comment]
Aha, thanks.
